src/main/java/com/infernostats/GodbookPlugin.java

```diff
@@ -74,11 +105,96 @@ public void onGameTick(GameTick event)
 		if (players.isEmpty())
 			return;
 
+		// Check if the localPlayer equipped a Lightbearer
+		if (config.lightbearerSupport() && players.containsKey(client.getLocalPlayer().getName())) {
+			Preach localPreach = players.get(client.getLocalPlayer().getName());
```

Comment:
`client.getLocalPlayer().getName()` can throw an error, do appropriate handling above this branch